src/main/groovy/com/ullink/Ikvm.groovy

```diff
@@ -204,6 +205,9 @@ class Ikvm extends ConventionTask {
         if (platform) {
             commandLineArgs += "-platform:${platform}"
         }
+        if (reference) {
```

Comment:
any reason not to use the supported list of references (through the configuration)? (maybe lack of documentation on how to use it ?) - see getReferences()